script-reference/types.rst

```diff
@@ -788,11 +788,29 @@ have no further structure.  An example declaration:
     type color: enum { Red, White, Blue, };
 
 The last comma after ``Blue`` is optional.  Both the type name ``color``
-and the individual values (``Red``, etc.) have global scope.
+and the individual values (``Red``, etc. -- not ``color::Red``) have
+global scope.
 
-Enumerations do not have associated values or ordering.
-The only operations allowed on enumerations are equality comparisons
-(``==``, ``!=``) and assignment (``=``).
+Enumerations may assign :zeek:type:`count` values explicitly:
```

Comment:
Note: there's not test for this currently.

Reply:
This test has them explicitly assigned: https://github.com/zeek/zeek/blob/master/testing/btest/bifs/enum_to_int.zeek

Reply:
Nothing in the BTest baselines currently has the error string if one uses e.g., a negative integer (`enumerator is not a count constant`). This is hard to test for in general though 🤷 .

Reply:
Ahh I see the distinction now, thanks